tests/functional/fixtures/groups.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+__all__ = ("group", "other_authority_group", "open_group", "user_owned_group")
+
+
+@pytest.fixture
+def group(factories, db_session):
+    # Readable by members, writable by members and joinable by authority
```

Comment:
Yeah, but I'm not sure these realistically can change, as huge numbers of tests would fail if they were altered. And if they did change I think we'd _want_ to fail in that case as it probably reflects a pretty major refactoring of the code base. You'd be fixing a lot of tests if you changed that stuff, so you'd probably want these ones to fail at that point too.

I could just remove them forcing the reader to go look them up in each case? At which point the comments could not be inaccurate, but it would leave the intention of the fixture unclear, so you couldn't work out what the author intended.

So do we want:

 * Repeat the default values here, meaning the functests can get out of step with the unit tests?
 * Comments which are accurate now, and are likely to be so, but could drift?
 * No comments meaning drift in meaning would be implicit rather than explicit?

I think it's sort of even's Steven in all directions. What do you think?

Reply:
Ok, Sean said it's dealer choice, so I'm going to leave the comments in. They'll act as a signpost of what was, if not what is.